odl/solvers/advanced/proximal_operators.py

```diff
@@ -369,15 +730,57 @@ def _call(self, x, out):
                 out.ufunc.maximum(lam, out=out)
 
                 # Global scaling
-                out /= lam
+                out /= (lam)
 
                 # Pointwise division
                 out.divide(diff, out)
 
     return _ProximalConvConjL1
 
 
-# TODO: move notes to ODL doc
+def proximal_l1(space, lam=1, g=None):
+    """Proximal operator factory of the l1-semi-norm.
```

Comment:
Yes. It's not the only such case :-)

On 2016-05-21 22:46, Jonas Adler wrote:
> In odl/solvers/advanced/proximal_operators.py
> <https://github.com/odlgroup/odl/pull/401#discussion_r64140037>:
> 
>>  
>>                  # Pointwise division
>>                  out.divide(diff, out)
>>  
>>      return _ProximalConvConjL1
>>  
>>  
>> -# TODO: move notes to ODL doc
>> +def proximal_l1(space, lam=1, g=None):
>> +    """Proximal operator factory of the l1-semi-norm.
> 
> Not my fault, was there before, removing i guess?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/odlgroup/odl/pull/401/files/5a721d1d1325be17940d8eeb33e35bdd4879ad8a#r64140037>
> 

-- 

Dr Holger Kohr
KTH - Royal Institute of Technology
Dpt of Mathematics
100 44 Stockholm
Tel +46 8 790-6652
Mobil +46 72 0119332